CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# How to Contribute
+
+### Running the app
+
+```
+$ npm install    # install required packages for creative collab
+$ npm start      # start creative collab at localhost:3000
+```
+
+## Finding things to contribute to
+
+Please use the [Issue Tracker](https://github.com/Mera-Gangapersaud/Creative-Collab/issues) to discuss
+potential improvements you want to make before sending a Pull Request.
+
+The Issue Tracker may contain items labelled [**help wanted**](https://github.com/Mera-Gangapersaud/Creative-Collab/labels/help%20wanted%20%F0%9F%99%8F%F0%9F%8F%BC) or [**good first bug**](https://github.com/Mera-Gangapersaud/Creative-Collab/labels/good%20first%20bug%20%F0%9F%90%9B)
+from time to time.
+
+## Pull Requests
+
+[Pull Requests](https://help.github.com/articles/about-pull-requests/) are used for contributions. Code must be well-tested and not decrease the test coverage significantly.
+
+#### Use a separate branch (not `master`)
+
+Please commit changes to a separate branch in your fork
+so that we can work together making changes to it before it
+is ready to be merged. Name your branch something like
```

Comment:
Oversight on my part, when an individual forks Creative Collab, it won't be `Mera-Gangapersaud/Creative-Collab` but `X/Creative-Collab`.

This means that users will be creating their branches using:

* git checkout -b username/issue-999

instead of:

* git checkout -b issue-999

Could we update this to:

`Branch names should follow the format of `issue-X`, where X is the issue number that your pull request fixes.`